### DISCUSSION TITLE
make CTRL-T work with folders outside the current one

### DISCUSSION THREAD

**Discussion by pogopaule** [junegunn/fzf#2791] [Q&A] [answered]
Let's assume I am in directory `/dirA`.
There is also a `/dirB`.

Very often there are case where I want to use CTRL-T to search relative to `dirB` although I am currently inside `dirA`.

For example
```
cd /dirA
...
cat /dirB/<CTRL-T>
```
So when there is a "path prefix" I'd like CTRL-T to search relative to that prefix.
When there is no prefix I'd like it to work relative to the current directory, like it already does now.

How on earth can I achieve that?

**Comment by pogopaule** [comment, marked as answer]
The answer can be found here https://github.com/junegunn/fzf/wiki/Configuring-fuzzy-completion#zsh

**Comment by Bellavene** [comment]
```
  --bind 'left:become(cd .. ;fd -a --exact-depth 1 | fzf)' \
  --bind 'right:become(cd {} 2>/dev/null;fd -a --exact-depth 1 | fzf)' \

```

**Reply by funnut** [reply]
Does become and cd work for actually changing the directory?

**Reply by nenahp** [reply]
`cd` is used in a subshell so it will not work in current shell process.

only work in your shell function
```
cd $(fzf --become ... )
```

**Comment by funnut** [comment]
I run ctrl t from files/ wherever I am
